src/main/kotlin/org/openrewrite/ListsOfRecipesWriter.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("SENSELESS_COMPARISON")
```

Comment:
We have to do stuff like this because IntelliJ yells at us that null checks are pointless - however, they aren't because TypeScript recipes don't guarantee non null in the same way that Java can. I ran into a lot of null pointer exceptions doing stuff that intellij said was unnecessary.